src/periodic.jl

```diff
@@ -1,8 +1,11 @@
 """
-    Periodic{T}(f, period::T, phase_shift::T) <: Transform
+    Periodic{P, S}(f, period::P, phase_shift::S) <: Transform
```

Comment:
```suggestion
    Periodic{P, S}(f, period::P, [phase_shift::S]) <: Transform
```

if `phase_shift` is optional this communicates it in the function signature, but it may not be necessary if you have a separate docstring for the other method.